data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/query/QueryRegistrationRequestsMGM.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "QueryRegistrationRequestsMGM",
```

Comment:
Do we need a new schema for this? Or can it be combined with `QueryRegistrationRequests`?

Reply:
I believe we already use `QueryRegistrationRequests` to allow the MGM to view registration requests.

Reply:
Combined with `QueryRegistrationRequests`